templates/out-of-sync.j2

```diff
@@ -106,9 +107,16 @@ $(document).ready(function () {
                 <span class="text-success" title="added">+{{added}}</span>&nbsp;<span class="text-danger" title="removed">-{{removed}}</span>
             </td>
             <td>{{ d.dependent_depth }}</td>
+            <td data-order="{{d.status.mathlib4_sync_prs | min}}">
+                <ul>
+                    {%- for pr in d.status.mathlib4_sync_prs %}
+                        <li><a href="https://github.com/leanprover-community/mathlib4/pull/{{pr}}">#{{pr}}</a></li>
+                    {%- endfor %}
+                </ul>
```

Comment:
Can you add a similar list near the top of the "Changes since the initial port" section of the `file.j2`?

Reply:
Done